lisa/tools/netperf.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Debian, Posix, Redhat, Suse
+from lisa.util import LisaException
+from lisa.util.process import Process
+
+from .git import Git
+from .make import Make
+
+
+class Netperf(Tool):
+    repo = "https://github.com/HewlettPackard/netperf/"
+    branch = "netperf-2.7.0"
+
+    @property
+    def command(self) -> str:
+        return "netperf"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def run_as_server(self, port: int = 30000, daemon: bool = True) -> None:
+        cmd = f"netserver -p {port} "
+        if not daemon:
+            cmd += " -D "
+        self.node.execute(
+            cmd,
+            shell=True,
+            expected_exit_code=0,
+            expected_exit_code_failure_message=f"fail to run {cmd}",
+        )
+
+    def run_as_client_async(
+        self,
+        server_ip: str,
+        core_count: int,
+        port: int = 30000,
+        test_name: str = "TCP_RR",
+        time: int = 400,
```

Comment:
does it mean 400 seconds? It's too long. Make it shorter.

Reply:
update to 150